group_vars/all.yml

```diff
@@ -2,7 +2,7 @@
 artifacts_repositories:
   enterprise:
     base_url: "https://artifacts.alfresco.com/nexus/content"
-    repository: "groups/private"
+    repository: "groups/internal"
```

Comment:
we don't want this on master

Reply:
because otherwise we can't test here -M versions, private doesn't contains them

Reply:
We need an action to specifically check no reference to internal repo is merged in master